packages/babel-generator/test/fixtures/types/ImportAttribute/input.js

```diff
@@ -0,0 +1 @@
+import foo from "foo.json" with type: "json";
```

Comment:
Can you also add a test case for dynamic import? Though currently it should be supported as well.
```js
import("foo.json", { with: { type: "json" } });
```